docs/install/any-kubernetes-cluster.md

```diff
@@ -219,6 +219,37 @@ The following commands install Istio and enable its Knative integration.
 
 {{< /tab >}}
 
+{{% tab name="Kong" %}}
+
+{{% feature-state version="v0.13" state="beta" %}}
+
+The following commands install Kong and enable its Knative integration.
+
+1. Install Kong Ingress Controller:
+
+   ```bash
+   kubectl apply --filename https://bit.ly/k4k8s
+   ```
+
+1. To configure Knative Serving to use Kourier by default:
+
+    ```bash
+    kubectl patch configmap/config-network \
+      --namespace knative-serving \
+      --type merge \
+      --patch '{"data":{"ingress.class":"kong"}}'
```

Comment:
Use `kong.ingress.networking.knative.dev` to be consistent what other providers currently use?

Reply:
The Ingress class for Knative Ingress resource and k8s Ingress resource are same in Kong's Ingress controller. This is to avoid users to think of two different ingress classes.

I do understand that this is a deviation from the other controllers that support Knative Ingress.